jax/_src/lax/lax.py

```diff
@@ -640,7 +641,8 @@ def dot(lhs: Array, rhs: Array, precision: PrecisionLike = None) -> Array:
                             Tuple[Sequence[int], Sequence[int]]]
 
 def dot_general(lhs: Array, rhs: Array, dimension_numbers: DotDimensionNumbers,
-                precision: PrecisionLike = None) -> Array:
+                precision: PrecisionLike = None,
+                preferred_element_type: Optional[DType] = None) -> Array:
```

Comment:
Please add this to the args section in the docstring.

Reply:
Done